src/main/java/model/piece/Byeong.java

```diff
@@ -0,0 +1,48 @@
+package model.piece;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+import model.Path;
+import model.Point;
+import model.Team;
+
+public class Byeong extends Piece {
+
+    public Byeong(Team team) {
+        super(team);
+        pieceName = PieceName.BYEONG;
+    }
+
+    @Override
+    public boolean isValidPoint(Point beforePoint, Point targetPoint) {
+        int moveForward = 1;
+
+        if (team == Team.RED) {
```

Comment:
team이 RED인지는 Team 에서 처리하는 것이 더 자연스럽겠네요 😄 

```suggestion
        if (team.isRed()) {
```

Reply:
오 이런 부분이 더 자연스러운 것 같네요!